src/components/Link/tests/Link.test.tsx

```diff
@@ -11,35 +11,36 @@ describe('onClick()', () => {
 });
 
 describe('external', () => {
-  it('sets external is false', () => {
-      const linkWrapper = mount(<Link external={false} />);
+  it('should verify the external is false', () => {
+      const linkWrapper = mount(<Link external={false}/>);
       expect(linkWrapper.prop('external')).toBe(false);
   });
 
-  it('sets external is true', () => {
-      const linkWrapper = mount(<Link external={true} />);
+  it('should verify the external is true', () => {
+      const linkWrapper = mount(<Link external={true}/>);
       expect(linkWrapper.prop('external')).toBe(true);
   });
 
-  it('sets the external type on the Link', () => {
+  it('should verify the external type on the Link', () => {
       const linkWrapper = mount(<Link external={true}/>).prop('external');
       expect(typeof linkWrapper).toBe('boolean');
```

Comment:
linkWrapper should have enzyme wrapper. Remove inline `.prop('external')` call.

Write as below
`expect(typeof linkWrapper.prop('external')).toBe('boolean'); `